clang/include/clang/Basic/DiagnosticCommonKinds.td

```diff
@@ -177,6 +177,10 @@ def warn_unknown_attribute_ignored : Warning<
   "unknown attribute %0 ignored">, InGroup<UnknownAttributes>;
 def warn_attribute_ignored : Warning<"%0 attribute ignored">,
   InGroup<IgnoredAttributes>;
+def warn_multiarg_malloc_attribute_ignored: Warning<
+  "'malloc' attribute ignored because"
+  " Clang does not support the one/two argument form">,
+  InGroup<IgnoredAttributes>;
```

Comment:
```suggestion
def warn_attribute_form_ignored : Warning<
  "%0 attribute ignored; Clang does not yet support this attribute signature">,
  InGroup<IgnoredAttributes>;
```
If we generalize the wording a bit, we can reuse this diagnostic for other circumstances where we don't yet support a particular signature.

Also, this diagnostic should be moved to DiagnosticSemaKinds.td (it's only diagnosed from `clang/lib/Sema`, so no need for it to be in the "common" diagnostics file for cross-lib diagnostics).

Reply:
Fixed in 02a153d9be1007b9603bcd14bf3f80d4bf900806, although I've very slightly changed the wording to match other warnings by using `because` instead of the `;`.

I've also added a couple of other error diagnostics in other commit, so please let me know if you think it's worth generalizing them too.